client/src/components/Common/Input/index.js

```diff
@@ -1,17 +1,27 @@
 import React from "react";
 import styled from "styled-components";
-import { spacing8 } from "../../../resources/style-constants";
+import { spacing16, theme1, grey1, grey2, spacing128, spacing8 } from "../../../resources/style-constants";
 
-const Wrapper = styled.div`
-  padding: ${spacing8} 0;
+const StyledInput = styled.input`
+    background-color: white;
+    margin: 0px;
+    padding: ${spacing16};
+    width: 40%;
+    border-radius: 10px;
+    border-width: 1px;
+    height: 20px;
+    font-size: 20px;
+    
 `;
 
-const Input = () => {
+
+const Input = ({type = "text", placeholder, onChange, fullwidth = false, ...props}) => {
   return (
-    <Wrapper>
-      <input type="text" />
-    </Wrapper>
+    <StyledInput placeholder = {placeholder} type = {type} onChange = {onChange} {...props}/>
```

Comment:
I know we did this part together, but you can make this even cleaner by condensing the code to { ...{ placeholder, type, onChange, fullWidth, ...props}}. 

Since all of the variable names match, you don't have to put placeholder = placeholder. Just as an example, if your onChange method was named changeHandler, then you would have to put onChange={changeHandler}